src/components/InstancedRigidBody.vue

```diff
@@ -0,0 +1,105 @@
+<script lang="ts" setup>
```

Comment:
In the editor I have "instanceRigidBody" (the wrong name, is my mistake) could you delete this file please

Reply:
`instanceRigidBody.vue` or `instancedRigidBody.vue`? (with `d` or without?)

Reply:
With a "d"

Reply:
Hum... so I keep the current file then,

It's `InstancedRigidBody.vue`

Reply:
Yes. Please delete mine